InstrumentationEngine.sln

```diff
@@ -92,535 +92,719 @@ Global
 	EndGlobalSection
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
 		CodeAnalysis.Managed|Any CPU = CodeAnalysis.Managed|Any CPU
+		CodeAnalysis.Managed|ARM64 = CodeAnalysis.Managed|ARM64
 		CodeAnalysis.Managed|x64 = CodeAnalysis.Managed|x64
 		CodeAnalysis.Managed|x86 = CodeAnalysis.Managed|x86
 		CodeAnalysis.Native|Any CPU = CodeAnalysis.Native|Any CPU
+		CodeAnalysis.Native|ARM64 = CodeAnalysis.Native|ARM64
 		CodeAnalysis.Native|x64 = CodeAnalysis.Native|x64
 		CodeAnalysis.Native|x86 = CodeAnalysis.Native|x86
 		Debug|Any CPU = Debug|Any CPU
+		Debug|ARM64 = Debug|ARM64
 		Debug|x64 = Debug|x64
 		Debug|x86 = Debug|x86
 		Release|Any CPU = Release|Any CPU
+		Release|ARM64 = Release|ARM64
 		Release|x64 = Release|x64
 		Release|x86 = Release|x86
 	EndGlobalSection
 	GlobalSection(ProjectConfigurationPlatforms) = postSolution
 		{DFC9E517-AF1A-4087-BB27-AD1A09602F4F}.CodeAnalysis.Managed|Any CPU.ActiveCfg = Release|Any CPU
 		{DFC9E517-AF1A-4087-BB27-AD1A09602F4F}.CodeAnalysis.Managed|Any CPU.Build.0 = Release|Any CPU
+		{DFC9E517-AF1A-4087-BB27-AD1A09602F4F}.CodeAnalysis.Managed|ARM64.ActiveCfg = Debug|Any CPU
+		{DFC9E517-AF1A-4087-BB27-AD1A09602F4F}.CodeAnalysis.Managed|ARM64.Build.0 = Debug|Any CPU
 		{DFC9E517-AF1A-4087-BB27-AD1A09602F4F}.CodeAnalysis.Managed|x64.ActiveCfg = Release|Any CPU
 		{DFC9E517-AF1A-4087-BB27-AD1A09602F4F}.CodeAnalysis.Managed|x86.ActiveCfg = Release|Any CPU
 		{DFC9E517-AF1A-4087-BB27-AD1A09602F4F}.CodeAnalysis.Native|Any CPU.ActiveCfg = Release|Any CPU
+		{DFC9E517-AF1A-4087-BB27-AD1A09602F4F}.CodeAnalysis.Native|ARM64.ActiveCfg = Debug|Any CPU
+		{DFC9E517-AF1A-4087-BB27-AD1A09602F4F}.CodeAnalysis.Native|ARM64.Build.0 = Debug|Any CPU
 		{DFC9E517-AF1A-4087-BB27-AD1A09602F4F}.CodeAnalysis.Native|x64.ActiveCfg = Release|Any CPU
 		{DFC9E517-AF1A-4087-BB27-AD1A09602F4F}.CodeAnalysis.Native|x86.ActiveCfg = Release|Any CPU
 		{DFC9E517-AF1A-4087-BB27-AD1A09602F4F}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
 		{DFC9E517-AF1A-4087-BB27-AD1A09602F4F}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{DFC9E517-AF1A-4087-BB27-AD1A09602F4F}.Debug|ARM64.ActiveCfg = Debug|Any CPU
 		{DFC9E517-AF1A-4087-BB27-AD1A09602F4F}.Debug|x64.ActiveCfg = Debug|Any CPU
 		{DFC9E517-AF1A-4087-BB27-AD1A09602F4F}.Debug|x86.ActiveCfg = Debug|Any CPU
 		{DFC9E517-AF1A-4087-BB27-AD1A09602F4F}.Release|Any CPU.ActiveCfg = Release|Any CPU
 		{DFC9E517-AF1A-4087-BB27-AD1A09602F4F}.Release|Any CPU.Build.0 = Release|Any CPU
+		{DFC9E517-AF1A-4087-BB27-AD1A09602F4F}.Release|ARM64.ActiveCfg = Release|Any CPU
 		{DFC9E517-AF1A-4087-BB27-AD1A09602F4F}.Release|x64.ActiveCfg = Release|Any CPU
 		{DFC9E517-AF1A-4087-BB27-AD1A09602F4F}.Release|x86.ActiveCfg = Release|Any CPU
 		{939151FA-8324-4197-B5B4-EAE927BD2AEE}.CodeAnalysis.Managed|Any CPU.ActiveCfg = Release|Win32
+		{939151FA-8324-4197-B5B4-EAE927BD2AEE}.CodeAnalysis.Managed|ARM64.ActiveCfg = Debug|x64
```

Comment:
This doesn't seem right.

Reply:
Good point, we should not be building the CodeAnalysis for ARM64 either, I'll fix this